frontend/src/components/notifications/NotificationsTab.vue

```diff
@@ -260,6 +272,10 @@ hr {
   font-weight: bold;
 }
 
+.subject-text {
+  font-size: 18px !important;
```

Comment:
!important should be avoided in css. If it's the only way it works for now we can revisit.

Reply:
Fixed